modulemd/modulemd-modulemetadata.c

```diff
@@ -1,6 +1,7 @@
 /* modulemd-metadata.c
  *
  * Copyright (C) 2017 Stephen Gallagher
+ * Copyright (C) 2017 Igor Gnatenko <ignatenkobrain@fedoraproject.org>
  *
```

Comment:
The code was just unreadable ;)

* there is no point in _free() function, no one ever should use such functions in GObject-based things, this will just lead in memory corruption at some point
* properties were renamed to match function names, because DESC vs set_description() is not really useful, also it wrongly translates to python property
* also I ordered code not alphabetically, but by most common properties (mdversion, name, stream, version, summary, description, ...)
* indentation and code style was inconsistent
* Probably something else I forgot about (IIRC there were trivial typos I fixed)

P.S. this was WIP patch, so commit messages might be not describing everything